src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -165,11 +165,17 @@
           "whatwg_encoding.js",
           "window.js");
 
-  /** Reserved words in ES6.1. Thanks to https://mathiasbynens.be/notes/reserved-keywords. */
+  /**
+   * Words for which we cannot generate 'namespace foo.Word {}'; Instead clutz tries to roll them up
+   * into properties of the parent namespace, but that can be lossy.
+   *
+   * <p>To regenerate this list use the following script with a list of candidates.
+   *
+   * <p>> for k in $(cat /tmp/keywords); do echo "namespace foo.$k {};" > /tmp/test.ts; \ > tsc
```

Comment:
If this comment is HTML i think your ">" prompts will be eaten, maybe switch to $?  (I also was confused by the random ">" in the middle of the command)